examples/framesjs-starter/app/examples/new-api-cache-control/frames/route.tsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable react/jsx-key */
+import { Button } from "@frames.js/next";
+import { createFrames } from "@frames.js/next";
+
+const frames = createFrames({
+  basePath: "/examples/new-api-cache-control",
+});
+
+const handleRequest = frames(async ({ message }) => {
```

Comment:
this example to needs to be linked to the from the examples page

Reply:
done https://github.com/framesjs/frames.js/pull/186/commits/8e8f92e246be9d3e93d2484b54ad3ea7f6ed5b6a